ssl/internal.h

```diff
@@ -3303,6 +3303,20 @@ struct SSL_CONFIG {
   // of support for AES hw. The value is only considered if |aes_hw_override| is
   // true.
   bool aes_hw_override_value : 1;
+
+  // conf_max_version_use_default indicates whether the |SSL_CONFIG| is configured
+  // to use the default maximum protocol version for the relevant protocol
+  // method. By default, |SSL_new| will set this to 0 and use that default.
```

Comment:
Won't it inherit from `SSL_CTX` where e.g. max version might not be 0? See https://github.com/aws/aws-lc/blob/main/ssl/ssl_lib.cc#L651-L652

Same for `conf_min_version_use_default`.

Reply:
Hm, this comment is poorly worded. Let me see if I can clarify.